package.json

```
@@ -41,7 +41,7 @@
     "translations": "docusaurus write-translations"
   },
   "dependencies": {
-    "@kisskissbankbank/kitten": "^4.14.1",
+    "@kisskissbankbank/kitten": "5.0.0",
```

```suggestion
    "@kisskissbankbank/kitten": "^5.0.0",
```

Pour le coup je ne fixerais pas pour que la lib prenne la version fixée par KissKiss